### PR TITLE
Adding my Iris-Rev4 and Lily58 keymaps

### PR DESCRIPTION
**Pull request by tehxeno**
## Description

I added my key configs for the:

- Keebio\iris\Rev4
- Lily58\Rev1

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/iris/keymaps/tehxeno/keymap.c [hunk 1]
```diff
@@ -0,0 +1,237 @@
+#include QMK_KEYBOARD_H
```
Comment (waffle87) [inline keyboards/keebio/iris/keymaps/tehxeno/keymap.c:1]: This file is missing a license header.

File: keyboards/keebio/iris/keymaps/tehxeno/keymap.c [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+
+#define _QWERTY 0
+#define _NAVI 1
+#define _NUMB 2
+#define _MOUSE 3
+#define _MEDIA 4
```
Comment (waffle87) [inline keyboards/keebio/iris/keymaps/tehxeno/keymap.c:7]:
```suggestion
enum layers {
    _QWERTY,
    _NAVI,
    _NUMB,
    _MOUSE,
    _MEDIA
};
```

File: keyboards/keebio/iris/keymaps/tehxeno/keymap.c [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+                is_alt_tab_active = true;
+                register_code(KC_LALT);
+              }
+              alt_tab_timer = timer_read();
+              tap_code16(S(KC_TAB));
+          }
+      }
+      break;
+  }
+  return true;
```
Comment (waffle87) [inline keyboards/keebio/iris/keymaps/tehxeno/keymap.c:226]:
```suggestion
  return false;
```

File: keyboards/lily58/keymaps/tehxeno/keymap.c [hunk 4]
```diff
@@ -0,0 +1,185 @@
+#include QMK_KEYBOARD_H
```
Comment (waffle87) [inline keyboards/lily58/keymaps/tehxeno/keymap.c:1]: This file is missing a license header.

File: keyboards/lily58/keymaps/tehxeno/keymap.c [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+
+#define _QWERTY 0
+#define _NAVI 1
+#define _NUMB 2
+#define _MOUSE 3
+#define _MEDIA 4
```
Comment (waffle87) [inline keyboards/lily58/keymaps/tehxeno/keymap.c:7]:
```suggestion
enum layers {
    _QWERTY,
    _NAVI,
    _NUMB,
    _MOUSE,
    _MEDIA
};
```

File: keyboards/lily58/keymaps/tehxeno/keymap.c [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+                                // └────────┴────────┴────────┘                 └────────┴────────┴────────┘
+  )
+};
+
+/* //These lines were commented out because I use layer numbers
+layer_state_t layer_state_set_user(layer_state_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
+*/
+
```
Comment (waffle87) [inline keyboards/lily58/keymaps/tehxeno/keymap.c:139]: The core tri layer feature should be used anyway.
```suggestion
```

File: keyboards/lily58/keymaps/tehxeno/config.h [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+#define RGBLIGHT_SAT_STEP 17
+#define RGBLIGHT_VAL_STEP 17
+
+// Underglow
+/*
+#undef RGBLED_NUM
+#define RGBLED_NUM 14    // Number of LEDs
+#define RGBLIGHT_ANIMATIONS
+#define RGBLIGHT_SLEEP
+*/
```
Comment (waffle87) [inline keyboards/lily58/keymaps/tehxeno/config.h:54]: Looks like RGB is disabled anyway.
```suggestion
```

**Comment by tehxeno** [comment]
Closing PR to work on changes. (Sorry if this isn't efficient, I'm not familiar with Git.)

**Pull request closed without merging**